netbox/templates/dcim/inc/devicebay.html

```diff
@@ -8,13 +8,17 @@
         <i class="fa fa-fw fa-{% if devicebay.installed_device %}dot-circle-o{% else %}circle-o{% endif %}"></i> {{ devicebay.name }}
     </td>
     {% if devicebay.installed_device %}
+        <td>
+            <span class="label label-{{ devicebay.installed_device.get_status_class }}">{{ devicebay.installed_device.get_status_display }}</span>
+        </td>
         <td>
             <a href="{% url 'dcim:device' pk=devicebay.installed_device.pk %}">{{ devicebay.installed_device }}</a>
         </td>
         <td>
             <span>{{ devicebay.installed_device.device_type.full_name }}</span>
         </td>
     {% else %}
+        <td></td>
         <td colspan="2">
```

Comment:
Lmk if it'd be better to span this cell three columns here instead of adding the spacer cell.